helm/reana/templates/roles.yaml

```diff
@@ -22,6 +22,16 @@ rules:
 - apiGroups: ["metrics.k8s.io"]
   resources: ["pods", "nodes"]
   verbs: ["get", "list", "watch"]
+# Custom dask kubernetes resources
```

Comment:
Also, when `dask.enabled` is true, it would be good to deploy Dask operator automatically, in quite the same way as we do for traefik; see the dependencies in `Chart.yaml`.